core/mobile/package.json

```diff
@@ -71,8 +71,7 @@
     "js-base64": "^3.6.1",
     "mocha": "^8.3.2",
     "rimraf": "^3.0.2",
-    "typescript": "~4.4.0",
-    "ws": "^7.5.3"
```

Comment:
any reason to remove the `ws` dependency here?

Reply:
it was already listed as a direct dep

Reply:
ah, missed that. Thanks